AstraEngine.Core/Entity.cs

```diff
@@ -269,6 +286,7 @@ public bool AddChild(Entity child)
     {
         if (_children.Add(child))
         {
+            child.Initialize();

```

Comment:
What do you do if the parent is not active when the child is added? What if the child being added is not active?